### ISSUE TITLE
Scrollable Inventory System

### ISSUE THREAD

**Issue by Jacki3** [LoGaCulture/LUTE#59]
**Is your feature request related to a problem? Please describe.**
We can increase the inventory size by using the inventory display script (this will increase the number of slots the inventory has) but if we have lots of items this display can fill the screen and make pressing on slots difficult! 

**Describe the solution you'd like**
Why not implement a scrollable pane of the set amount of slots that fit into the same space as the default inventory?

**Additional context**
An easy imlpementation - just need to add scroll rects to the areas that require it! The tricky aspect will be to setup the logic behind it in code (but this should be feasible).

**Comment by Jacki3** [comment]
Adding scroll rect and mask will work but 2 things:

1. the inventory display script seems to alter changes made when entering play mode
2. using the keyboard to go up/down will not scroll (can be modified in code and not too pertainable as we are mostly mobile based)
